docker-compose.yml

```diff
@@ -14,3 +14,4 @@ services:
 
   redis:
```

Comment:
This service can be renamed to redis-stack-server and then there's is no need to specify the `container_name`